tests/api.c

```diff
@@ -54815,6 +54834,154 @@ static int test_wolfSSL_dtls_set_mtu(void)
     return 0;
 }
 
+#if defined(HAVE_IO_TESTS_DEPENDENCIES) && !defined(SINGLE_THREADED) && \
+    defined(WOLFSSL_DTLS)
+
+static WC_INLINE void generateDTLSMsg(byte* out, int outSz, word32 seq,
+        enum HandShakeType hsType)
+{
+    size_t idx = 0;
+    byte* l;
+
+    /* record layer */
+    /* handshake type */
+    out[idx++] = handshake;
+    /* protocol version */
+    out[idx++] = 0xfe;
+    out[idx++] = 0xfd; /* DTLS 1.2 */
+    /* epoch 0 */
+    XMEMSET(out + idx, 0, 2);
+    idx += 2;
+    /* sequence number */
+    XMEMSET(out + idx, 0, 6);
+    c32toa(seq, out + idx + 2);
+    idx += 6;
+    /* length in BE */
+    c16toa(outSz - idx - 2, out + idx);
+    idx += 2;
+
+    /* handshake layer */
+    /* handshake type */
+    out[idx++] = (byte)hsType;
+    /* length */
+    l = out + idx;
+    idx += 3;
+    /* message seq */
+    c16toa(0, out + idx);
+    idx += 2;
+    /* frag offset */
+    c32to24(0, out + idx);
+    idx += 3;
+    /* frag length */
+    c32to24(outSz - idx - 3, l);
```

Comment:
```Testing: ./configure --enable-all
Running: make check
make[2]: warning: -j5 forced in submake: resetting jobserver mode.
tests/api.c:54876:25: error: implicit conversion loses integer precision: 'unsigned long' to 'word32' (aka 'unsigned int') [-Werror,-Wshorten-64-to-32]
    c32to24(outSz - idx - 3, l);
    ~~~~~~~ ~~~~~~~~~~~~^~~
tests/api.c:54877:25: error: implicit conversion loses integer precision: 'unsigned long' to 'word32' (aka 'unsigned int') [-Werror,-Wshorten-64-to-32]
    c32to24(outSz - idx - 3, out + idx);
    ~~~~~~~ ~~~~~~~~~~~~^~~
2 errors generated.```

Reply:
Should be fixed. Let's see what Jenkins has to say.